src/get_navbar.js

```diff
@@ -0,0 +1,6 @@
+let MongoClient = require('mongodb').MongoClient;
```

Comment:
Looks like this isn't used. Should be removed?

Reply:
Yes, the entire file can be deleted